neofetch

```diff
@@ -1522,7 +1529,7 @@ get_memory() {
 }
 
 get_song() {
-    player="$(ps -e | grep -m 1 -o -F \
+    player="$(ps -e | grep -m 1 -o \
```

Comment:
I want to keep `-F` if possible. If we need regex we should benchmark the speed difference between the two.

Reply:
Well, regex is only needed for the iTunes match, because it needs to see the `iTunes` binary running rather than `iTunesHelper`.  Without it, Spotify will never get detected.  Possibly a separate check for macOS?